vnext/PropertySheets/x86.props

```diff
@@ -9,7 +9,7 @@
 
   <ItemDefinitionGroup>
     <ClCompile>
-      <CallingConvention>StdCall</CallingConvention>
+      <CallingConvention>Cdecl</CallingConvention>
```

Comment:
>Cdecl [](start = 25, length = 5)

If we do this, then do we need to add __cdecl?

Reply:
So we can export these headers and consuming projects are not forced to use `__cdecl`.